File: backend/OlivePlatform/OlivePlatform.Application/Features/Plots/Requests/PlotsRequestFilter.cs
using OlivePlatform.Application.Common;
using OlivePlatform.Domain.Entities;

public class PlotsRequestFilter : PaginationRequest
{
    public string? Reference { get; set; }

    public string? Name { get; set; }

    public OliveVariety? OliveVariety { get; set; }
}

File: frontend/olive-platform/packages/core/src/features/plots/domain/entities/PlotsRequestFilter.ts
import type { PaginationFilter } from "../../../../core/PaginationFilter";
import type { OliveVarieties } from "../../../shared/entities/OliveVarieties";

export type  PlotsRequestFilter = PaginationFilter &  {
reference?: string,
name?: string,
oliveVariety?: OliveVarieties
} 